scheduler/src/cook/scheduler/constraints.clj

```diff
@@ -108,14 +108,14 @@
   (job-constraint-evaluate
     [{:keys [job]} _ vm-attributes _]
     (let [k8s-vm? (= (get vm-attributes "compute-cluster-type") "kubernetes")
-          job-gpu-count-requested (-> job util/job-ent->resources :gpus (or 0))]
+          job-gpu-count-requested (-> job util/job-ent->resources :gpus (or 0) int)]
```

Comment:
Might also be nice to explain why we need the `int` here

Reply:
add